File: 1-iniciante/1012/1012.lua
PI = 3.14159
A, B, C = io.read("*n","*n","*n")
print(
	"TRIANGULO: " .. string.format("%.3f", A * C / 2) .. "\n" ..
	"CIRCULO: " .. string.format("%.3f", PI * C^2) .. "\n" ..
	"TRAPEZIO: " .. string.format("%.3f", (A + B) * C / 2) .. "\n" ..
	"QUADRADO: " .. string.format("%.3f", B^2) .. "\n" ..
	"RETANGULO: " .. string.format("%.3f", A * B)
)

File: 1-iniciante/1012/1012.cs
using System;

class URI {
    static void Main(string[] args) {
        double A, B, C, PI = 3.14159;
        string[] input;
        input = Console.ReadLine().Split();
        A = double.Parse(input[0]);
        B = double.Parse(input[1]);
        C = double.Parse(input[2]);
        Console.WriteLine($"TRIANGULO: {A * C / 2:0.000}");
        Console.WriteLine($"CIRCULO: {C * C * PI:0.000}");
        Console.WriteLine($"TRAPEZIO: {(A + B) * C / 2:0.000}");
        Console.WriteLine($"QUADRADO: {B * B:0.000}");
        Console.WriteLine($"RETANGULO: {A * B:0.000}");
    }
}